.gitignore

```diff
@@ -268,4 +268,11 @@ Session.vim
 config.ps1
 **/IISApplications
 
+### source-build ###
 prebuilt/
+msbuild.binlog
+
+# Exists temporarily during build. Interferes with VersionTools'
+# UpdateDependenciesAndSubmitPullRequest: if no upgrades are performed, the
+# sanity check fails when "git status" shows this file.
+msbuild.ProjectImports.zip
```

Comment:
Interesting I've never seen this file. Is there a way to get it redirected to bin/obj instead?

Reply:
Looks like it's placed in the same folder as the `binlog` file, so I can change `/bl` to `/bl:bin/logs/msbuild.binlog` to make it show up in `"bin\logs\output.ProjectImports.zip`, or `/bl:bin/obj/msbuild.binlog`.